megameklab/src/megameklab/printing/InventoryWriter.java

```diff
@@ -699,7 +704,7 @@ private double printEquipmentTable(List<? extends InventoryEntry> list,
                         case DAMAGE:
                             lines = Math.max(lines, sheet.addMultilineTextElement(canvas, colX[i],
                                     yPosition,
-                                    colX[i + 1] - colX[i] - fontSize, lineHeight, line.getDamageField(row),
+                                    colX[i + 1] - colX[i] - 4, lineHeight, line.getDamageField(row),
                                     fontSize, SVGConstants.SVG_START_VALUE, SVGConstants.SVG_NORMAL_VALUE));
```

Comment:
If `fontSize` is in reference to something that is adjustable or even static, it needs to stick to being a variable or a static value. Not just a number. We need to know WHAT that number represents.